internal/common/utils.go

```diff
@@ -11,6 +11,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
```

Comment:
FYI we usually put the std lib packages at the top followed by 3rd parties

Reply:
Updated with new commit.